src/utils/utils.h

```diff
@@ -50,7 +50,10 @@ using BytesArr = std::array<Byte, N>;
 using BytesArrView = std::span<const Byte, std::dynamic_extent>;
 using BytesArrMutableView = std::span<Byte, std::dynamic_extent>;
 
-/// ethCallInfo: tuple of (from, to, gasLimit, gasPrice, value, functor, data)
+/**
+ * ethCallInfo: tuple of (from, to, gasLimit, gasPrice, value, functor, data)
+ * ...
```

Comment:
I think I specified this poorly now that I realize it, the `...` was meant to be "the rest of the comment lines below it". My bad, just fixed it myself with 998c843.